changelog.md

```diff
@@ -1,5 +1,10 @@
 ## Change Log
 
+### [v0.6.0](https://github.com/wyze/bs-react-testing-library/releases/tag/v0.6.0) (2019-07-05)
```

Comment:
Revert this as it is done when published to npm.

Reply:
Done.